### ISSUE TITLE
Parallel checks

### ISSUE THREAD

**Issue by Falci** [htools-org/htools-sitecheck#3]
Wondering if the main block of checks could run in parallel, maybe even split in multiple endpoints/call so the UI could display their status independently 

https://github.com/htools-org/htools-sitecheck/blob/0571030ee166b7198eef3f8e5b44f587167b746a/lib/index.js#L62-L88

**Comment by rithvikvibhu** [comment]
True! Not everything since a few checks depend on previous ones, but yeah most can be in parallel.

I've been meaning to rewrite this and make a better UI with other checks for stateless DANE, etc. and in general show more info so that there's no need for a cli or HNS resolver to debug anything:
- Show TLS certificate
- Website screenshot preview
- Show dig +trace output
- Show all on-chain records, when they were updated last, etc.

And like you said, with this much info, it's better to break it up into different API endpoints.
